modules/phase_field/include/kernels/ParamReaction.h

```diff
@@ -0,0 +1,35 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef PARAMREACTION_H
+#define PARAMREACTION_H
+
+#include "Kernel.h"
+
+// Forward Declarations
+class ParamReaction;
```

Comment:
We prefix kernels that add a constant coefficient with` Coeff` (CoefTimeDerivative CoefDiffusion CoefCoupledTimeDerivative CoefDiffusion) and we do have `CoeffReaction` in MOOSE test. I propose we move that one to the framework, or at least rename this one to the same name.